_data/simple-icons.json

```diff
@@ -1337,6 +1337,11 @@
             "hex": "551DEF",
             "source": "https://byte.co/byte"
         },
+        {
+            "title": "ByteDance",
+            "hex": "3D8BFF",
```

Comment:
I think I would opt for `#3C8CFF` as used on their website. What do you think?

Reply:
Yes. #3C8CFF is great. I will update it. Again, there are something wrong with [Roy Picker](https://www.useroy.com/) (Wrong color palette setup or some filter are using on website or etc), I think I will compare with source code 😆 bugs in my workflow.